bcda/service/service_test.go

```diff
@@ -225,7 +228,7 @@ func (s *ServiceTestSuite) TestIncludeSuppressedBeneficiaries() {
 	}
 }
 
-func (s *ServiceTestSuite) TestGetNewAndExistingBeneficiaries() {
+func (s *ServiceTestSuite) TestGetNewAndExistingBeneficiariesIntegration() {
```

Comment:
There are 2-3 tests that I had to make minor updates to; because we are filtering by the ACO model now, the configured test CMS ID of `cmsID` was not being parsed and these tests also weren't loading the config files to do the parsing.